lib/app/views/guided.erb

```diff
@@ -69,6 +69,9 @@
         <% elsif error_code == 'no_redirect' %>
           The redirect URI for this instance was visited, but no sequence was actively running that was waiting for this event.  Please run the a launch sequence test
           before visiting this redirect url.
+        <% elsif error_code == 'ehr_launch' %>
+          No actively running launch sequences found for iss <%=instance.url%>.
```

Comment:
This isn't quite correct.  The `iss` is the URI being passed as a query parameter from the auth server, which could be different than what we have in our instance (and hence the error).  You need to pass that along in the querystring  so you can grab it here.  For example, `error_code=ehr_launch&iss=params[:iss]`.  Though you may want to doublecheck that the iss that gets passed in the querystring is URI encoded/escaped properly.

Reply:
Think it's updated to what you wanted, fixed iss and added checking for waiting state